cartridge/roles/metrics.lua

```diff
@@ -255,4 +264,5 @@ return setmetatable({
     apply_config = apply_config,
     set_export = set_export,
     set_default_labels = set_default_labels,
+    set_is_health_handler = set_is_health_handler,
```

Comment:
Since this section is called `'health'` in clusterwide config, I think it's fine to call it just `set_health_handler` here.